src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -68,23 +71,49 @@ const UserInformationContent: React.FunctionComponent<PropsType> = ({
 
   if (!user || !userWithId) return null;
 
+  const handlerChangeStatus = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    const { value } = e.target;
+    makeUpdateUserOnlineStatus({
+      userUid: userWithId?.id,
+    })(value);
+  };
+
+  const statuses = ["online", "busy", "offline", "custom"];
```

Comment:
This should be implemented as an enum

Reply:
And not within this component here, but in a types/* location or similar.